marklogic-data-hub-central/ui/e2e/cypress/e2e/explore/jsonTableViewValidations.cy.tsx

```diff
@@ -201,28 +201,19 @@ describe("json scenario for table on browse documents page", () => {
     browsePage.getTableView().should("have.css", "color", "rgb(57, 68, 148)");
   });
 
-  // TODO: DHFPROD-10183
-  it.skip("Search for multiple facets, switch to snippet view, delete a facet, switch to table view, verify search query", () => {
+  it("Search for multiple facets, switch to snippet view, delete a facet, switch to table view, verify search query", () => {
     entitiesSidebar.openBaseEntityDropdown();
     entitiesSidebar.selectBaseEntityOption("Customer");
     entitiesSidebar.getBaseEntityOption("Customer").should("be.visible");
     browsePage.waitForSpinnerToDisappear();
-
-    browsePage.getFacet("Adams Cole").trigger("mouseover");
-    cy.wait(1000);
-    browsePage.getTooltip("Adams Cole").should("not.exist");
-    browsePage.getFacetItemCheckbox("name", "Adams Cole").click();
-    browsePage.getFacet("adamscole@nutralab.com").trigger("mouseover");
-    cy.wait(1000);
-    browsePage.getTooltip("adamscole\\@nutralab\\.com").should("be.exist");
```

Comment:
Why are we skipping this tooltip check? Are we handling it in another test?

Reply:
@aniamy I did some check tooltip  in graph nodes some weeks ago but I don't remember where.

Reply:
That specific tooltip is not anymore in the webapp